frame/ranked-collective/src/lib.rs

```diff
@@ -635,6 +637,19 @@ pub mod pallet {
 				pays_fee: Pays::No,
 			})
 		}
+
+		// Only root account can add a member to
+		#[pallet::weight(100)]
+		pub fn add_member_to_rank(
```

Comment:
There are unrelated changes in this MR.  
Reviews go a lot faster if you tidy it up a bit first to make it minimal.

Reply:
Am pushing another PR which is clean and small for specific issue. These PR's commits got messed up